front/tests/003-study-management.spec.ts

```diff
@@ -1,148 +1,132 @@
+
+test.describe('Validate the Study creation workflow', () => {
+  let project: Project;
+  let study: Study;
+  let OSRDLanguage: string;
+  test.beforeAll(' Retrieve a project', async () => {
+    project = await getProject();
+  });
 
-test.beforeEach(async () => {
-  study = await postApiRequest(`/api/projects/${project.id}/studies/`, {
-    ...studyData,
-    name: `${studyData.name} ${uuidv4()}`,
-    budget: 1234567890,
-    project: project.id,
+  test.beforeEach('Create a new study linked to the project', async ({ page }) => {
+    const homePage = new HomePage(page);
+    await homePage.goToHomePage();
+    OSRDLanguage = await homePage.getOSRDLanguage();
   });
-});
 
-test.describe('Test if study creation workflow is working properly', () => {
+  /** *************** Test 1 **************** */
   test('Create a new study', async ({ page }) => {
     const studyPage = new StudyPage(page);
-    const commonPage = new CommonPage(page);
-
-    await page.goto(`/operational-studies/projects/${project.id}`);
-
-    await studyPage.openStudyCreationModal();
-
-    const studyName = `${studyData.name} ${uuidv4()}`;
-    await studyPage.setStudyName(studyName);
-
-    await studyPage.setStudyTypeByText(studyData.type);
-
-    await studyPage.setStudyStatusByText(studyData.state);
-
-    await studyPage.setStudyDescription(studyData.description);
-
-    const todayDateISO = new Date().toISOString().split('T')[0];
-    await studyPage.setStudyStartDate(todayDateISO);
-
-    await studyPage.setStudyEstimatedEndDate(todayDateISO);
-
-    await studyPage.setStudyEndDate(todayDateISO);
-
-    await studyPage.setStudyServiceCode(studyData.service_code);
-
-    await studyPage.setStudyBusinessCode(studyData.business_code);
-
-    await studyPage.setStudyBudget(studyData.budget);
-
-    await commonPage.setTag(project.tags[0]);
-    await commonPage.setTag(project.tags[1]);
-    await commonPage.setTag(project.tags[2]);
-
-    await studyPage.createStudyButton.click();
-
-    await expect(studyPage.studyEditionModal).not.toBeVisible();
-
-    await expect(studyPage.getStudyName).toHaveText(studyName);
-    await expect(studyPage.getStudyDescription).toHaveText(studyData.description);
-    await expect(studyPage.getStudyType).toHaveText(studyData.type);
-    await expect(studyPage.getStudyState.first()).toHaveText(studyData.state);
-
-    await expect(studyPage.studyServiceCodeInfo).toHaveText(studyData.service_code);
-    await expect(studyPage.studyBusinessCodeInfo).toHaveText(studyData.business_code);
-    expect(await studyPage.getNumericFinancialsAmount()).toBe('1 234 567 890 €');
-    const tags = await studyPage.getStudyTags.textContent();
-    expect(tags).toContain(studyData.tags.join(''));
+    await page.goto(`/operational-studies/projects/${project.id}`); // Navigate to project page
+
+    // Set translations based on the language
+    const translations = OSRDLanguage === 'English' ? enTranslations : frTranslations;
```

Comment:
could be a util function

Reply:
A utility function would be useful, however, the primary challenge is that the structures of `enTranslations` and `frTranslations` vary each time they are called. This variation means that the parsing logic would differ based on the specific translation structure, making a universal function complex and hard to maintain.

Reply:
I’ve explored several solutions, and while one approach works, it introduces unnecessary complexity to the code, which could impact readability and maintainability.

Reply:
ok np